packages/blocks/timer/src/app.tsx

```diff
@@ -1,106 +1,293 @@
-import React, { useEffect, useState, useCallback } from "react";
+import "./app.scss";
+
+import React, {
+  useState,
+  useCallback,
+  useMemo,
+  useRef,
+  MouseEventHandler,
+} from "react";
 import { BlockComponent } from "blockprotocol/react";
+import { parseISO, isValid } from "date-fns";
+import * as duration from "duration-fns";
+import { useAutoRefresh } from "./app/use-auto-refresh";
+import { calculateDurationStepLength } from "./app/calculate-duration-step-length";
+import { DurationInput } from "./app/duration-input";
+import { TimerStatus } from "./app/timer-status";
+
+type TimerState = {
+  /** https://schema.org/Duration */
```

Comment:
I think https://en.wikipedia.org/wiki/ISO_8601#Durations is probably more helpful here, as that's where you are going to end up going via schema.org to find out what the format is.

Reply:
Fixed in https://github.com/hashintel/hash/pull/576/commits/0f1cc9b8698bbaf13c0410e92cb02877d5332bfd